src/index.js

```diff
@@ -1,37 +1,141 @@
 import "./style.css";
+// import { toDoTask } from "./modules/todo.js";
+
+import Store from "./modules/Store.js";
+import toDoTask from "./modules/todo.js";
+import { check } from "./modules/check.js";
+// class toDoTask {
+//   constructor(desc, index, completed = false) {
+//     this.desc = desc;
+//     this.index = index;
+//     this.completed = completed;
+//   }
+// }
+
+//start ui class
+
```

Comment:
- [ ] Kindly remove all the commented code to clean up the codebase.

Reply:
done